arangod/GeoIndex/Index.cpp

```diff
@@ -114,8 +114,9 @@ Result Index::indexCells(VPackSlice const& doc, std::vector<S2CellId>& cells,
     S2LatLng ll = S2LatLng::FromDegrees(lat.getNumericValue<double>(),
                                         lon.getNumericValue<double>());
     if (!ll.is_valid()) {
-      return TRI_ERROR_QUERY_INVALID_GEO_VALUE;
-    }
+      LOG_TOPIC(DEBUG, arangodb::Logger::FIXME)
```

Comment:
should this log messafe happen higher up in the call chain ? We do not log in all the other bad parameter case I think